src/app/components/EmbedConsentBanner/index.tsx

```diff
@@ -3,12 +3,15 @@ import { SocialEmbedProviders } from '../../models/types/global';
 
 import ConsentBanner from './ConsentBanner';
 
-const CONSENT_BANNER_PROVIDERS: SocialEmbedProviders[] = ['youtube'];
+export const CONSENT_BANNER_PROVIDERS: Extract<
+  SocialEmbedProviders,
+  'youtube' | 'tiktok'
+>[] = ['youtube', 'tiktok'];
```

Comment:
[Extract](https://www.typescriptlang.org/docs/handbook/utility-types.html#extracttype-union) is kinda similar to `Pick` but for 'unions', whereby you pick from a type the values you want. In this case, we are using the `SocialEmbedProviders` union type, which includes all of our social media providers. I just want `youtube` and `tiktok` from that type so that the `EmbedConsentBanner` will only be compiled when supplying those 2 social media providers.

This `const` is then exported so that it can be used in the other sub-components to lock in the types for them to only those 2 social media providers.